tests/functional/botocore/test_models_directory.py

```diff
@@ -24,10 +24,31 @@ def _available_services():
     return botocore.session.Session().get_available_services()
 
 
-def test_models_contains_no_examples_files(loader):
-    assert loader.list_available_services('examples-1') == []
+@pytest.mark.parametrize(
+        "type_name",
+        ['api-2', 'docs-2', 'examples-1', 'resources-1', 'smoke', 'smoke-2']
+)
+def test_models_contain_no_unknown_file_types(type_name, loader):
```

Comment:
Could we make this an allow list instead? Mainly I'm concerned if a new JSON model is introduced that we do not want to include it will not be caught by this list.